### ISSUE TITLE
Column widths are adjusted for vertical scrollbar width even if it is not present/necessary

### ISSUE THREAD

**Issue by pan-henryk** [Swimlane/angular-data-table#159]
I noticed that column width are adjusted for vertical scroll bar width even if it is disabled (scrollbarV = false) or unnecessary (item count smaller than paging size)
It becomes noticeable if you want to add grid (bottom borders) to table rows.

**Comment by amcdnl** [comment]
Just need to wrap this line in a check for that property. https://github.com/Swimlane/angular-data-table/blob/master/src/components/DataTableDirective.js#L76

Can you PR plz?

**Comment by pan-henryk** [comment]
Yes, but will address only half of the problem. If we have scrollbarV = true and count < pagingSize (that is all dat fits one page, no overflow), the problem will still exist. I tried to fix it, but it seems column widths are calculated before we know how much data we have?
